src/qoi/include/grins/rayfire_mesh.h

```diff
@@ -39,24 +39,40 @@ namespace GRINS
     
   //! RayfireMesh
   /*!
-  This class creates a 1D mesh of EDGE2 elements to allow
-  for integration of functions along a line across the domain
+  This class performs a rayfire across a given mesh.
+  The user supplies an origin point on the mesh boundary,
+  and spherical angle(s) theta (and phi for 3D) to describe the direction of the rayfire.
```

Comment:
\theta is the angle from the x-axis in the x-y plane? \phi is the angle from the x-y plane with positive being in the positive half of the z-plane? If so, please indicate. Otherwise, please specify what the angles are.